src/enums.ts

```diff
@@ -1,6 +1,7 @@
 enum BbbHookEvents {
   Update = 'UPDATE_HOOK',
   NewSubscriber = 'NEW_SUBSCRIBER',
+  Unsubscribe = 'UNSUBSCRIBE'
```

Comment:
Please use the same naming pattern for the two options.

If subscribe event is "NEW_SUBSCRIBER" the unsubscribe should be "REMOVED_SUBSCRIBER"